src/test/resources/features/completion/after_pep_object_completion.feature

```diff
@@ -0,0 +1,127 @@
+Feature: Completion after pep object
+
+  Scenario: Complete at toplevel
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    pep.
+    """
+    When I put the caret after pep.
+    And I invoke autocompletion popup
+    Then completion list should contain:
+      | __init__                   |
```

Comment:
No need to mention all methods that could be completed here. I suggest to choose some key methods reasonable from user point of view, not everyting including private methods (named `_*` or `__*`)

Reply:
Please fix this and all other tests

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/424/commits/dcaea2a9a74dff8d433706ce1fff2d61620318b4